src/SFA.DAS.ApprenticeCommitments.Web/Services/ApprenticeAccountProvider.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Threading.Tasks;
+using SFA.DAS.ApprenticePortal.Authentication;
+
+namespace SFA.DAS.ApprenticeCommitments.Web.Services
+{
+    public class ApprenticeAccountProvider : IApprenticeAccountProvider
+    {
+        private readonly ApprenticeApi _client;
+
+        public ApprenticeAccountProvider(ApprenticeApi client)
+        {
+            _client = client;
+        }
+
+        public async Task<IApprenticeAccount?> GetApprenticeAccount(Guid id)
+        {
+            return await _client.TryGetApprentice(id);
```

Comment:
Does the `TryGetApprentice` method take care of interpreting 404 responses as not found (rather than error)?  I can't remember off hand.

Reply:
yes it does